loginrequest.php

```diff
@@ -9,7 +9,7 @@
 		//Connect.php (tells where to connect servername, username, password, dbaseName)
 		require_once('connect.php');
 		//username and password sent from form
-		$myusername = mysqli_real_escape_string($con, $_POST['username']);
+		$myusername = ucwords(mysqli_real_escape_string($con, $_POST['username']));

```

Comment:
Ensures that Graham can log in properly as the database is not case sensitive but the checks to see if the user is Graham are case sensitive